### PR TITLE
CORE-1215: Remove WKHandlersExport.h

### PR DESCRIPTION
**Pull request by EBGToo**
Push the currency-specific WKHandlers into .../src/walletkit/handlers/.../WKHandler<Currency>.c

### CODE REVIEW

File: WalletKitCore/src/walletkit/WKConfig.h [hunk 1]
```diff
@@ -269,19 +282,21 @@ DEFINE_MODES            ("tezos-testnet", WK_SYNC_MODE_API_ONLY)
 DEFINE_NETWORK (WK_NETWORK_TYPE_XLM,  "stellar-mainnet", NETWORK_NAME, "mainnet", true, 35516170, 1, 5)
 DEFINE_NETWORK_FEE_ESTIMATE ("stellar-mainnet", "100", "5s", 5 * 1000)
 DEFINE_CURRENCY ("stellar-mainnet",     "stellar-mainnet:__native__",   NETWORK_NAME,  WK_NETWORK_CURRENCY_XLM,  "native",   NULL,   true)
-DEFINE_UNIT ("stellar-mainnet:__native__",  "lumen_i",     "xlm_i",   0,  "XLM_I")
-DEFINE_UNIT ("stellar-mainnet:__native__",  "lumen",       "xlm",     7,  "XLM")
+    DEFINE_UNIT ("stellar-mainnet:__native__",  "lumen_i",     "xlm_i",   0,  "XLM_I")
+    DEFINE_UNIT ("stellar-mainnet:__native__",  "lumen",       "xlm",     7,  "XLM")
```
Comment (bgoring) [inline WalletKitCore/src/walletkit/WKConfig.h:286]: No change?

**Review by bgoring — APPROVED** [review]
Clever. So to add a new set of handlers:
1. identify it in WkConfig.h with empty DEFINE_HANDLERS for the network byte
2. Change the cmakefiles.txt to add the handler c (which would be part of any new handler group anyways)
3. Create a WKHandlers<networktype>.c within the new handler group that identifies the handlers

File: WalletKitCore/src/walletkit/WKConfig.h (hunk 1 above)
Comment (EBGToo) [reply WalletKitCore/src/walletkit/WKConfig.h:286]: Spacing consistent w/ other currencies; indentation implies the Unit's Currency.

**Comment by EBGToo** [comment]
> Clever. So to add a new set of handlers:
> 
> 1. identify it in WkConfig.h with empty DEFINE_HANDLERS for the network byte
> 2. Change the cmakefiles.txt to add the handler c (which would be part of any new handler group anyways)
> 3. Create a WKHandlers.c within the new handler group that identifies the handlers

Right.  The change attempts to centralize everything needed to add a new currency.  You did the 'account' work; this PR does the 'handlers' work (which I realized during your 'account' work!).

There is still `enum WKNetworkType` - but I think that can't use `WKConfig.h`.

**Pull request merged**